test/fixtures/test-runner/output/abort_hooks.snapshot

```diff
@@ -101,7 +101,7 @@ not ok 2 - 2 after describe
         *
         *
         *
-        async Promise.all (index 0)
```

Comment:
nit: do you think losing the index of the failing hook could impact DevEx?

Reply:
I don't think this is the index of the hook. But no, I don't think this internal stack frame is important to DX.